services/account-without-juggler/repositories/account/datasources/mysqlconn.ts

```diff
@@ -0,0 +1,201 @@
+const debug = require("debug")("account-without-juggler");
+const mysql = require("mysql");
+const db = require("mysql-promise")();
+import { CrudConnector } from "loopback-next/packages/repository/src/crud-connector";
+import { IError } from "@types/mysql";
+import {
+  Class,
+  DataSource,
+  Entity,
+  EntityData,
+  Filter,
+  ObjectType,
+  Options,
+  Where
+} from "loopback-next/packages/repository";
+
+export class MySqlConn implements CrudConnector {
+  private connection: any;
+
+  constructor(config: Object) {
+    db.configure(config, require("mysql2"));
```

Comment:
That is not what our module (loopback-connector-mysql) uses though. Should we not test this with the same driver that our module currently uses?